presto-main/src/main/java/io/prestosql/sql/planner/LogicalPlanner.java

```diff
@@ -387,10 +390,18 @@ private RelationPlan createInsertPlan(Analysis analysis, Insert insertStatement)
         String catalogName = insert.getTarget().getCatalogName().getCatalogName();
         TableStatisticsMetadata statisticsMetadata = metadata.getStatisticsCollectionMetadataForWrite(session, catalogName, tableMetadata.getMetadata());
 
+        List<String> visibleTableColumnNames = insertedColumns.stream()
```

Comment:
- `visibleTableColumnNames` is collected only for the purpose to be streamed again
- "visible*" is a wrong name, since this is "inserted"

```
List<ColumnHandle> insertedColumnHandles = insertedColumns.stream()
  .map(ColumnMetadata::getName)
  .map(columns::get)
  .collect(toImmutableList()));
```